boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -181,11 +181,21 @@
 	};
 };
 
-&spi3 {
-	status = "okay";
-	sck-pin = <41>;
-	mosi-pin = <42>;
-	miso-pin = <43>;
+arduino_serial: &uart1 {
+	status = "disabled";
+	compatible = "nordic,nrf-uart";
+	current-speed = <115200>;
+	rx-pin = <33>;
+	tx-pin = <34>;
+};
+
+arduino_i2c: &i2c0 {};
```

Comment:
`i2c0` node properties are modified above, just add `arduino_i2c: ` there instead of doing this.